BeeTracker/app/src/main/res/layout/sign_up_fragment.xml

```diff
@@ -130,26 +131,27 @@
             android:textColor="@color/black"
             android:textColorHint="@color/lightGrey"
             android:textSize="@dimen/login_signup_input_text_size"
+            android:nextFocusDown="@+id/sign_up_password_input"
             android:theme="@style/LoginSignupEditTextTheme" />
 
         <TextView
-            android:id="@+id/password_label"
+            android:id="@+id/sign_up_password_label"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:layout_alignParentStart="true"
-            android:layout_below="@id/email_input"
+            android:layout_below="@id/sign_up_email_input"
             android:layout_marginTop="@dimen/medium_small_padding"
-            android:labelFor="@+id/password_input"
+            android:labelFor="@+id/sign_up_password_input"
```

Comment:
Oh ok, just read up on what the difference between @+id vs @id is. I think for consistency, it would be good to have the id definition of an object to have the @+id even if it's not the first occurrence in the file. And then all the labelFor or layout_below properties can refer to those objects using @id.

Reply:
Makes sense